### ISSUE TITLE
Annotations that target annotations: named entities, georeference, etc

### ISSUE THREAD

**Issue by jacobwegner** [jhu-digital-manuscripts/AnIOp#90]
(Branched from #89)

@jabrah wrote:

> Based on using our georeference annotations in the M3 plugin, there is some feedback I have about these kinds of annotations. I think it would be helpful for a web annotation viewer to have annotations that target other annotations to explicitly call out the targeted annotation in its target property. Not sure how possible this is. The current plugin parses the CTS URN to guess at the intended target annotation, which may not always be correct.

> For example, a georeference annotation that targets some translated text:

```
{
  "type": "SpecificResource",
  "source": {
    "type": "Annotation",
    "id": "https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:35r/translation-alignment/622/text/"
  },
  "selector": {
    "type": "TextQuoteSelector",
    "exact": "Athens"
  }
}
```

**Comment by jacobwegner** [comment]
@jabrah  The current plugin parses the CTS URN to guess at the intended target annotation, which may not always be correct

I'll take another look at the URNs in the Georeference WA.

Part of the issue is that the subreferences from that WA are targeting a different translation ("Barber", [perseus-eng4:](https://scaife.perseus.org/reader/urn:cts:greekLit:tlg0012.tlg001.perseus-eng4:1.1/)) than the translation that is in the translation alignment WA ("Murray", [perseus-eng3:](https://scaife.perseus.org/reader/urn:cts:greekLit:tlg0012.tlg001.perseus-eng3:1.1/)).

The CITE URNs we're using to key off the translation alignments assume the Greek Venetus A transcription aligned to the Murray English translation, so it is possible that a place mentioned in the Murray translation may not exist verbatim in the barber (or not within the same text part designated by the URN).

As far as adding the new targets you're suggesting, I think for the named entities we're providing (e.g. https://aniop-atlas-staging.eldarion.com/wa/urn:cite2:hmt:msA.v1:12r/named-entities/collection/compound/0/) we could take one of two approaches

1. We can use the translation alignment WA (because the Greek content matches the sub-references we're providing in the existing target.
2. We could create an annotation for _just_ the Greek content within a given folio, and use that as the target.

On the Georeference WA, we could fake "#1" above (because of the slight differences between the Barber and Murray translations) or we could create an annotation for _just_ the Barber content within a given folio.

**Comment by jacobwegner** [comment]
@jabrah One other note...I had been thinking about dropping the `text` version of the translation alignment annotation (as discussed on #89).

If we're going to use the `TextQuoteSelector`, would that work well enough for you with the HTML version of the translation alignment annotation?

**Comment by jabrah** [comment]
I don't know about the content itself, so it's useful to have these sorts of insights,

* At the moment, my preference would be for targeting the translation alignment web annotations, since this would be directly usable by the plugin without interpretation, given that it currently renders the translation alignment annotations.
  * There could be some interesting interactions bringing in a single annotation with the Greek content, then perhaps having the translation alignment annotations targeting that, along with named entities, etc. Probably out of scope though :) 
* `TextQuoteSelector` would work fine with the HTML version

**Comment by jacobwegner** [comment]
> There could be some interesting interactions bringing in a single annotation with the Greek content, then perhaps having the translation alignment annotations targeting that, along with named entities, etc. Probably out of scope though :)

I agree it'd be nice to have, but out of scope; we'd really need a new alignment for the Barber english translation to the Greek for that to work.

I'll work on updating our alignments as you've suggested this week:

- [ ] Add targets to the Translation Alignment WA
- [ ] Add [**TextQuoteSelector**](https://www.w3.org/TR/annotation-model/#text-quote-selector)s

**Comment by jacobwegner** [comment]
@jabrah as an update, I wasn't able to get to the target and TextQuoteSelector changes this week, but plan on picking it back up next week.

(Unlike #89, there shouldn't be any backwards incompatible changes)

**Comment by jacobwegner** [comment]
@jabrah I haven't made it to the changes discussed here.  Would they still be helpful to you?

**Comment by jabrah** [comment]
In future development, but not immediately, I would be comfortable deferring this until later
